types/src/traits/network.rs

```diff
@@ -173,8 +193,9 @@ pub trait NetworkMsg:
 /// exposes low level API for interacting with a network
 /// intended to be implemented for libp2p, the centralized server,
 /// and memory network
+// TODO ED Uppercae these?
```

Comment:
Nit: curious about this TODO. The trait name looks good, so maybe something else?